### PR TITLE
Add LOOKin integration

### PR DESCRIPTION
**Pull request by ANMalko**
## Proposed change

[LOOKin](https://look-in.club/en/devices) focuses on providing devices that integrate with a local api and can be used internet free.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/58125 https://github.com/home-assistant/core/pull/58175
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2907
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by bdraco** [comment]
This looks good now https://deploy-preview-19884--home-assistant-docs.netlify.app/integrations/lookin/.  We do need a logo in the brands PR before we can merge though.

File: source/_integrations/lookin.markdown [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: LOOKin
+description: Instructions on how to integrate LookIN devices into Home Assistant.
+ha_category:
+  - Sensor
```
Comment (bdraco) [inline source/_integrations/lookin.markdown:6]:
```suggestion
  - Climate
  - Sensor
```

File: source/_integrations/lookin.markdown [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ha_release: 2021.11
+ha_iot_class: Local Push
+ha_codeowners:
+  - '@ANMalko'
+ha_domain: lookin
+ha_config_flow: true
+ha_platforms:
+  - sensor
```
Comment (bdraco) [inline source/_integrations/lookin.markdown:15]:
```suggestion
  - climate
  - sensor
```

**Review by frenck — APPROVED** [review]
Thanks, @ANMalko 👍

**Pull request merged**